src/style/components/asset/file-asset.less

```diff
@@ -22,7 +22,7 @@
   width: 100%;
   flex-wrap: wrap;
   align-self: center;
-  padding: 12px;
+  padding: 12px 0 12px 12px;
```

Comment:
Since the general designer's idea was to introduce additional whitespace at the right of the conversation, I guess we don't need the right padding of the message bodies